### DISCUSSION TITLE
nvim setup

### DISCUSSION THREAD

**Discussion by deveezy** [fitrh/init.nvim#9] [General]
Hi, can you help with your nvim setup config? I want to remap telescope.find_files to leader + ff, but this mapping not working for the first time, if i press for example leader + e to open file browser and after this action i press leader + ff again then it works

**Comment by fitrh** [comment]
You need to update its `packerspec` file, more precisely, this line: https://github.com/fitrh/init.nvim/blob/c41865c0c44180aee13c98ffbc76de6604331a95/lua/config/plugin/telescope/packerspec.lua#L20
Add your keymap there, then, recompile packer

**Reply by deveezy** [reply]
It works! Thx for amazing config :)
